components/root_layout/root_layout.vue

```diff
@@ -161,17 +155,103 @@ export default {
   },
 
   computed: {
-    responsiveClass () {
-      if (!this.responsiveBreakpoint) return;
-      return `d-root-layout__responsive--${this.responsiveBreakpoint}`;
-    },
-
-    bodyClasses () {
-      return [
-        this.bodyClass,
-        { 'd-root-layout__body--invert': this.sidebarPosition === ROOT_LAYOUT_SIDEBAR_POSITIONS.RIGHT },
-      ];
+    isInverted () {
+      return this.sidebarPosition === ROOT_LAYOUT_SIDEBAR_POSITIONS.RIGHT;
     },
   },
 };
 </script>
+
+<style lang="less">
+.d-root-layout {
+  position: relative;
+  display: grid;
+  grid-template-areas:
+    'header header'
+    'sidebar body'
+    'footer footer';
+  grid-template-columns: min-content 1fr;
+  min-height: 100vh;
+
+  &--inverted {
+    grid-template-areas:
+      'header header'
+      'body sidebar'
+      'footer footer';
+    grid-template-columns: 1fr min-content;
+  }
+
+  &--fixed {
+    height: 100vh;
+  }
+
+  &__header {
+    grid-area: header;
+
+    &--sticky {
+      position: sticky;
+      top: 0;
+      z-index: var(--zi-base1);
+    }
+  }
+
+  &__sidebar {
+    grid-area: sidebar;
+    height: 100%;
+    box-shadow: none;
+
+    &--sticky {
+      position: sticky;
+      top: 0;
+      overflow: auto;
+    }
```

Comment:
I cannot find the usage of this class

Reply:
hmmm was the sidebar supposed to stay in place when sticky is enabled? can't even remember what it did before

Reply:
I checked an earlier version and seemed it does not. So I think the current functionality is fine and this class is not needed anymore.

Reply:
Removed.